.secrets.baseline

```diff
@@ -115,7 +115,8 @@
       "pattern": [
         "(?i)dummyapikey",
         "(?i)test-secret",
-        "^x-api-key$"
+        "^x-api-key$",
+        "^1f9d73167e2166b707c6$"
```

Comment:
This secret is within a multi-line string so I couldn't use an inline exclusion comment next to it